lib/OrdinaryDiffEqNonlinearSolve/test/newton_tests.jl

```diff
@@ -1,4 +1,4 @@
-using OrdinaryDiffEqCore
+using OrdinaryDiffEq
```

Comment:
why was this changed?

Reply:
Not sure. I haven't made any change to it.